core.js

```diff
@@ -2761,7 +2761,190 @@ HTMLAudioElement.HAVE_METADATA = HTMLMediaElement.HAVE_METADATA;
 HTMLAudioElement.HAVE_CURRENT_DATA = HTMLMediaElement.HAVE_CURRENT_DATA;
 HTMLAudioElement.HAVE_FUTURE_DATA = HTMLMediaElement.HAVE_FUTURE_DATA;
 HTMLAudioElement.HAVE_ENOUGH_DATA = HTMLMediaElement.HAVE_ENOUGH_DATA;
+class Video {
+  static getDevices() {
+    return [];
+  }
+}
+
+class MediaDeviceKind {
+}
+MediaDeviceKind.audioInput = "audioinput";
```

Comment:
Looks like these don't exist yet, so is there a point to this enum for now?

Reply:
It's from the spec.

Reply:
The spec says it's a class?

Reply:
The spec says it's an enum, but that's not a thing in JS. We can redefine it as a plain object if you want.

Reply:
Yeah I think the spec means it as a more abstract concept. I don't think that implies it should be constructible as a class.

I prefer the Object approach 👍.

Reply:
Done